tests/test_section.py

```diff
@@ -11,8 +11,12 @@
 from docx.enum.section import WD_ORIENT, WD_SECTION
 from docx.section import Section, Sections
 from docx.shared import Inches
+from docx.header import Header
 
 from .unitutil.cxml import element, xml
+from .unitutil.mock import (
+    instance_mock, class_mock
+)
```

Comment:
This can all go on one line. I only enclose them in parens when they overflow a 78 character line.